stdlib/argparse.pyi

```diff
@@ -43,14 +43,14 @@ _ActionStr: TypeAlias = str
 # callers that don't use a literal argument
 _NArgsStr: TypeAlias = str
 
-ONE_OR_MORE: Literal["+"]
-OPTIONAL: Literal["?"]
-PARSER: Literal["A..."]
-REMAINDER: Literal["..."]
+ONE_OR_MORE: Final = "+"
+OPTIONAL: Final = "?"
+PARSER: Final = "A..."
+REMAINDER: Final = "..."
 _SUPPRESS_T = NewType("_SUPPRESS_T", str)
-SUPPRESS: _SUPPRESS_T | str  # not using Literal because argparse sometimes compares SUPPRESS with is
+SUPPRESS: Final[_SUPPRESS_T | str]  # not using Literal because argparse sometimes compares SUPPRESS with is
```

Comment:
Let's keep this PR limited to uncontroversial changes of switching `Literal` uses to `Final`. This change doesn't fit that description, and the comment on this line makes me think that we'd need to look at it closely before making a decision on it.

Reply:
Good catch, definitely an accident having this be in this PR: https://github.com/python/typeshed/pull/12332/commits/64e75eb49417fcb8d1b624f15f9e4433748142fb